docs/troubleshoot/openid-connect.md

```diff
@@ -0,0 +1,118 @@
+---
+layout: default
+title: Troubleshoot OpenID Connect
+parent: Troubleshoot
+nav_order: 3
+---
+
+# OpenID Connect troubleshooting
+
+This page includes troubleshooting steps for using OpenID Connect with the Security plugin.
+
+
+---
+
+#### Table of contents
+- TOC
+{:toc}
+
+
+---
+
+## Set log level to debug
+
+To help troubleshoot OpenID Connect, set the log level to `debug` on Elasticsearch. Add the following lines in `config/log4j2.properties` and restart the node:
+
+```
+logger.opendistro_security.name = com.amazon.dlic.auth.http.jwt
+logger.opendistro_security.level = trace
+```
```

Comment:
Hey @hardik-k-shah, I could use some help on the correct strings for these logging settings. I wasn't able to dig them up in code.

Reply:
Looks good.